Add Backup Documentation (#2830)
kyma-project/kyma: docs/backup/docs/01-01-backup.md

---
title: Overview
---

The backup in Kyma uses [Ark](https://github.com/heptio/velero/).

> >**NOTE:** The Ark project is now called Velero, as it will be migrated when the new Velero version is available.

Reviewer: ```suggestion
>**NOTE:** The Ark project was renamed to Velero. The full migration is planned once the new Velero version is available.
```

---
title: Overview
---

> Backup in Kyma uses [Ark](https://github.com/heptio/velero/) (The Project is renamed to Velero. As soon as there is a new Velero Version, Ark will be migrated to Velero). Ark is a tool to backup kubernetes resources and store them in a Blob storage. Beside that, Ark is triggering Physical Volumen Snapshots and stores the snapshot references as part of the Backup. It can create backups on demand or on schedule, filter objects which should be backed up, and set TTL (time to live) for stored backups.

Author: I would like to keep the velero explenation as we are linking sometimes already to velero documentation. Just to make sure peope are not confused.

> Backup in Kyma uses [Ark](https://github.com/heptio/velero/) (The Project is renamed to Velero. As soon as there is a new Velero Version, Ark will be migrated to Velero). Ark is a tool to backup kubernetes resources and store them in a Blob storage. Beside that, Ark is triggering Physical Volumen Snapshots and stores the snapshot references as part of the Backup. It can create backups on demand or on schedule, filter objects which should be backed up, and set TTL (time to live) for stored backups.

Reviewer: Added the note.
